author/script/perlimports

```diff
@@ -70,6 +70,21 @@ to ignore. (See above for behaviour). The pattern is one module name per line.
     Foo
     Foo::Bar
 
+=head2 --ignore-modules-pattern
+
+A comma-separated list of module names which should be ignored by this script.
```

Comment:
I think this wording needs a tweak, since it's a list of regexes rather than purely a list of module names. Also, it looks like the implementation doesn't split on commas. We could make this a repeatable arg, though and avoid trying to split. That would allow multiple regexes in both the pattern arg and in the filename.

> The type spec may end in @ to indicate that the option may appear multiple times.

https://metacpan.org/pod/Getopt::Long::Descriptive#@opt_spec

Reply:
I think it's so easy to use parenthesis for regexi that not sure it's worth it?

Reply:
Sure. Whatever you think is best. It's easy enough to change later. 😀